apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-palette/dot-palette-contentlets/dot-palette-contentlets.component.html

```diff
@@ -0,0 +1,46 @@
+<dot-palette-input-filter
+    #filterInput
+    [goBackBtn]="true"
+    (goBack)="showContentTypesList()"
+    (filter)="filterContentlets($event)"
+></dot-palette-input-filter>
+
+<div class="dot-content-palette__items" *ngIf="items?.length; else Loading">
+    <div
+        *ngFor="let item of items"
+        draggable="true"
+        data-testId="paletteItem"
+        (dragstart)="dragStart(item)"
+    >
+        <img src="/dA/{{ item.inode }}/titleImage/48w" *ngIf="item?.hasTitleImage === true" />
```

Comment:
ngIf else with the `dot-contentlet-icon`

Reply:
for such a simple html, I won't be implementing that change, cuz it requires https://angular.io/api/common/NgTemplateOutlet and that will make it harder to read/understand